app/views/policies/privacy.md

```diff
@@ -43,18 +43,20 @@ We collect this information to perform our contract with you (e.g. to provide th
-* Datadog to aggregate metrics and application performance data. Some personal information may be stored on their servers. Their use of this information is subject to the [Data Processing Addendum](https://www.datadoghq.com/legal/data-processing-addendum/) between Datadog and Ruby Central and the Datadog [Privacy Policy](https://www.datadoghq.com/legal/privacy/).  
-* Honeycomb.io to aggregate metrics and application performance data. Only information related to web traffic is stored on their servers. Their use of this information is subject to the [Data Processing Addendum](https://www.honeycomb.io/dpa) between Honeycomb and Ruby Central and the Honeycomb [Privacy Notice](https://www.honeycomb.io/privacy).    
-* Honeybadger to monitor, track, and resolve application errors.  Their use of this information is subject to the [Data Processing Addendum](https://www.honeybadger.io/terms/data-processing-addendum/) between Honeybadger and Ruby Central and the Honeybadger [Privacy Statement](https://www.honeybadger.io/privacy/).  
-* Twilio to send emails through the SendGrid product. Their use of this information is subject to the [Data Protection Addendum](https://www.twilio.com/en-us/legal/data-protection-addendum) between Twilio and Ruby Central and the Twilio [Privacy Notice](https://www.twilio.com/en-us/legal/privacy).
+* Datadog to aggregate metrics and application performance data. Some personal information may be stored on their servers. Their use of this information is subject to the [Data Processing Addendum](https://www.datadoghq.com/legal/data-processing-addendum/) between Datadog and Ruby Central.  
+* Honeycomb.io to aggregate metrics and application performance data. Only information related to web traffic is stored on their servers. Their use of this information is subject to the [Data Processing Addendum](https://www.honeycomb.io/dpa) between Honeycomb and Ruby Central.    
+* Honeybadger to monitor, track, and resolve application errors.  Their use of this information is subject to the [Data Processing Addendum](https://www.honeybadger.io/terms/data-processing-addendum/) between Honeybadger and Ruby Central.  
+* Twilio to send emails through the SendGrid product. Their use of this information is subject to the [Data Protection Addendum](https://www.twilio.com/en-us/legal/data-protection-addendum) between Twilio and Ruby Central.
 
+We also have a partnership with ClickHouse to enable retrieval and analysis of historic RubyGems download log data, and to make some log data publicly available to the Ruby community. The data we share with ClickHouse includes geolocation data, which we use for internal analysis of RubyGems usage, but the only location data we make publicly available is continent and country from which downloads originate. ClickHouse’s use of private geolocation data is subject to the [Data Processing Addendum](https://clickhouse.com/legal/agreements/data-processing-addendum) between ClickHouse and Ruby Central.
```

Comment:
Per my understanding both occurences of RubyGems should be replaced with RubyGems.org in this part.

Reply:
I've updated the doc with additional rubygems.org references.